src/content/administration/private-repositories/ssh-key.mdx

```diff
@@ -20,14 +20,6 @@ The public SSH is available on the `Settings` section on the `Admin -> General`
 
 <p align="center"><Image src="/docs/administration/private-repositories/images/private-repository-ssh-key.png" alt="show SSH key" width="676" /></p>
 
-### Retrive the SSH key using kubectl <TiersList tiers="Self-Hosted" />
```

Comment:
@pchico83 in which version is this available? if it's coming on 0.14, can you merge this on the 0.14 branch?

Reply:
And can you add how to do it via the UI?

Reply:
@rberrelleza this is available from many versions ago, the feature has at least 1 year. The UI flow is right above the section I am removing